x-pack/plugins/security/public/management/role_mappings/components/no_compatible_realms/no_compatible_realms.tsx

```diff
@@ -19,15 +19,15 @@ export const NoCompatibleRealms: React.FunctionComponent = () => {
       title={
         <FormattedMessage
           id="xpack.security.management.roleMappings.noCompatibleRealmsErrorTitle"
-          defaultMessage="No compatible realms are enabled in Elasticsearch"
+          defaultMessage="No compatible realms appear to be enabled in Elasticsearch"
```

Comment:
This wasn't necessary, but it softens the language because we aren't 100% sure that there aren't any compatible realms